src/styles.scss

```diff
@@ -184,6 +191,14 @@ body {
   line-height: 28px !important;
 }
 
+.mat-icon-\[17px\] {
+  margin-top: 2px;
```

Comment:
this is a trap. wherever you use the `mat-icon-[17px]` class you should add a `mt-[2px]` class rather than sneaking the 2px top margin in here (violates naming matters)